sklearn/linear_model/coordinate_descent.py

```diff
@@ -786,19 +786,19 @@ class Lasso(ElasticNet):
 
     Parameters
     ----------
-    alpha : float, optional
-        Constant that multiplies the L1 term. Defaults to 1.0.
+    alpha : float, default 1.0.
```

Comment:
we're not consistent about the formatting. Sometimes it's

alpha : float, optional (default 1.0)

if we make such commits we should have a clear policy and a test would be a good idea.

See :

https://github.com/mne-tools/mne-python/blob/master/mne/tests/test_docstring_parameters.py

we use this in mne-python to make sure all parameters are documented and in the right order.